### PR TITLE
remove margin-top: 2rem; in Swage

### PR DESCRIPTION
**Pull request by Tealk**
Closes #

Changes proposed in this pull request:

- remove `margin-top: 2rem;` see screenshot

How to test the feature manually:

test in Browser Developer Tools

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/edge/docs/en/developers/04_Pull_requests.md).

**Screenshots**
Removed the yellow bar
image

### CODE REVIEW

**Comment by Alkarex** [comment]
Please try to run:

```sh
make fix-all
make test-all
```

**Comment by Tealk** [comment]
i can only do that at home, i don't have make, composer and whatever else will be needed on my work computer.

**Comment by Tealk** [comment]
I don't quite understand this now, why does the command want to access bin/composer?
```
$ make fix-all
bin/composer run-script fix
make: bin/composer: No such file or directory
make: *** [Makefile:203: composer-fix] Error 127
```

```
$ composer self-update
You are already using the latest available Composer version 2.4.1 (stable channel).
```

and i have to install a complete php environment?
```
$ make test-all
mkdir -p bin/
wget 'https://raw.githubusercontent.com/composer/getcomposer.org/76a7060ccb93902cd7576b67264ad91c8a2700e2/web/installer' -O - -q | php -- --quiet --install-dir='./bin/' --filename='composer'
bin/composer install --prefer-dist --no-progress
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Your lock file does not contain a compatible set of packages. Please run composer update.

  Problem 1
    - Root composer.json requires PHP extension ext-gmp * but it is missing from your system. Install or enable PHP's gmp extension.
  Problem 2
    - Root composer.json requires PHP extension ext-intl * but it is missing from your system. Install or enable PHP's intl extension.
  Problem 3
    - Root composer.json requires PHP extension ext-pdo_sqlite * but it is missing from your system. Install or enable PHP's pdo_sqlite extension.
  Problem 4
    - Root composer.json requires PHP extension ext-zend-opcache * but it is missing from your system. Install or enable PHP's zend-opcache extension.

To enable extensions, verify that they are enabled in your .ini files:
    - /etc/php/php.ini
You can also run `php --ini` in a terminal to see which files are used by PHP in CLI mode.
Alternatively, you can run Composer with `--ignore-platform-req=ext-gmp --ignore-platform-req=ext-intl --ignore-platform-req=ext-pdo_sqlite --ignore-platform-req=ext-zend-opcache` to temporarily ignore these required extensions.
make: *** [Makefile:102: vendor/bin/phpstan] Error 2
```

**Comment by Alkarex** [comment]
Ah, the required PHP extensions are due to https://github.com/FreshRSS/FreshRSS/pull/4497 . We could try to ignore the few of them not relevant for the tests (so far - because if our tests were more exhaustive, all would be needed). But those extensions are needed anyway to run/test FreshRSS manually.

Besides that and php + node/npm, our Makefile indeed installs what it needs in `./bin/`

I am not sure why you got the first error `make: bin/composer: No such file or directory`. Maybe PHP was not installed at all?

**Comment by Tealk** [comment]
hmm i would have to build some of those extensions from the aur :(

**Comment by Alkarex** [comment]
I am also often mounting the source code inside one of our Docker images to try with various PHP environments.
What platform are you using?

Example:

```sh
cd ./FreshRSS/

docker run --rm -v $PWD:/var/www/FreshRSS freshrss/freshrss:edge-alpine bin/composer run-script fix
docker run --rm -v $PWD:/usr/src/app -w /usr/src/app node:lts-alpine npm run fix

docker run --rm -v $PWD:/var/www/FreshRSS freshrss/freshrss:edge-alpine bin/composer test
docker run --rm -v $PWD:/usr/src/app -w /usr/src/app node:lts-alpine npm test
```

**Comment by Tealk** [comment]
hmm docker is not running for me right now, I'll have to look at it again tomorrow

File: p/themes/Swage/swage.css.map [hunk 1]
```diff
@@ -1 +1 @@
...
...,iBAAA,CAGD,cACC,SAAA,CACA,YAAA,CAKD,sBACC,kBA71CW,CA81CX,aAAA,CACA,WAAA,CACA,gBAAA,CACA,iBAAA,CACA,kBAAA,CAEA,4BACC,wBAAA,CAGD,4BACC,oBAAA,CAKH,UACC,WAAA,CAEA,eACC,QAAA,CAGD,iBACC,QAAA,CAEA,sBACC,QAAA,CAIF,uBACC,OAAA,CACA,gBAAA,CACA,wBAAA,CACA,iBAAA,CAEA,4BACC,aAAA,CAGD,6BACC,UAAA,CAKH,2CAEC,kBA/4CY,CAg5CZ,aAAA,CACA,WAAA,CACA,gBAAA,CACA,iBAAA,CACA,kBAAA,CAEA,uDACC,wBAAA,CAIF,oGAEC,YAAA,CAGD,sBACC,QAAA,CACA,SAAA,CAGD,WACC,gBAAA,CAGD,cACC,UAAA,CAEA,sBACC,wBAAA,CACA,aAAA,CACA,MAAA,CAEA,4BACC,UAAA,CAGD,4BACC,YAAA,CAKH,aACC,qBAAA,CAGD,MACC,iBAAA,CACA,kBAAA,CAGD,WACC,cAAA,CAGD,mBACC,iBAAA,CACA,KAAA,CAGD,uFAKC,iBAAA,CAGD,oBACC,UAAA,CACA,KAAA,CAGD,2BAEC,OAAA,CAGD,YACC,KAAA,CAGD,uDAGC,QAAA,CAGD,qBACC,QAAA,CAGD,mBACC,UAAA,CAAA,CAIF,yBACC,iBACC,QAAA,CAAA,CAIF,yBACC,gBACC,YAAA,CAAA,CAIF,eACC,YAAA,CAGD,6CACC,kBAAA,CAGD,uCACC,kBAAA,CACA,gBAAA,CAIA,iBACC,aA5gDY,CA8gDZ,wBACC,kBA/gDW,CAghDX,aAnhDW,CAqhDX,0BACC,aAthDU,CA0hDZ,mBACC,aAxhDW,CA4hDb,iBACC,aAAA,CAEA,wBACC,kBAAA,CACA,aApiDW,CAsiDX,0BACC,aAviDU,CA2iDZ,mBACC,aAAA,CAKH,qBACC,gBAAA,CACA,kBAnjDa,CAojDb,aAljDa,CAmjDb,WAAA,CAEA,6BACC,eAAA,CACA,aAAA,CACA,aAAA,CAKD,mCACC,MAAA,CACA,UAAA,CAKD,oCACC,OAAA,CACA,SAAA,CAKD,cACC,kBAAA","file":"swage.css"}
```
Comment (math-GH) [inline p/themes/Swage/swage.css.map]: I do not have knowledge about `.map`. I am not sure if it make sense to remove `sourceRoot`. Could you please support us with more information?

Comment (Tealk) [reply p/themes/Swage/swage.css.map]: I use this extension: https://github.com/glenn2223/vscode-live-sass-compiler
I haven't dealt with the map files that much either, but so far it has always worked very well, with no errors in the browser console and the w3 tests haven't given any error messages either.

Comment (math-GH) [reply p/themes/Swage/swage.css.map]: I use https://marketplace.visualstudio.com/items?itemName=yutent.scss-to-css, that generates the CSS in a different style

**Comment by Tealk** [comment]
> I am also often mounting the source code inside one of our Docker images to try with various PHP environments. What platform are you using?

how should the output actually be with the command?
```
$ docker run --rm -v $PWD:/var/www/FreshRSS freshrss/freshrss:edge-alpine bin/composer run-script fix
> cli/manipulate.translation.php -a format
> phpcbf . -p -s
sh: phpcbf: not found
Script phpcbf . -p -s handling the phpcbf event returned with error code 127
Script @phpcbf was called via fix
```

**Comment by Alkarex** [comment]
That Docker command still requires that you have the required tools in your local `./bin/` folder. This can be done automatically when calling make (maybe we should have a `make install-dev` or something to do only that) or you can manually pick from the Makefile the relevant parts. Or we could maybe provide a specific Dockerfile for tests.

**Review by math-GH — APPROVED** [review]
It is fine for me.
(the css.map file: I do not have a clue if it is an improvement or it will break something. For me it works)

**Pull request merged**